administrator/language/en-GB/com_users.ini

```
@@ -314,7 +309,42 @@ COM_USERS_SUBMENU_NOTES="User Notes"
 COM_USERS_SUBMENU_NOTE_CATEGORIES="User Note Categories"
 COM_USERS_SUBMENU_USERS="Users"
 COM_USERS_TFA_ACTIVE="Uses Two Factor Authentication"
+COM_USERS_TFA_ADD_AUTHENTICATOR_OF_TYPE="Add a new %s"
+COM_USERS_TFA_ADD_PAGE_HEAD="Add a Two Factor Authentication method"
```

```suggestion
COM_USERS_TFA_ADD_PAGE_HEAD="Add a Two Factor Authentication Method"
```

language style guide